src/fetch_business_partners_info.py

```diff
@@ -0,0 +1,104 @@
+import os
+import json
+import datetime
+import time
+import pandas as pd
+from unicodedata import normalize
+from selenium import webdriver
+from bs4 import BeautifulSoup
+
+
+def selenium_driver_path():
+    for path in ['chromedriver.exe', 'chromedriver']:
```

Comment:
> This adds decencies to the project that should be considered in the dev environment set up the process. I'm not a Selenium expert, but maybe we might expand this list so it will work with any of the major browsers, not only Chrome. Does that make sense?

Yeah, this makes sense and also I'll figure out how to download the driver in order to setup the environment automatically. But I'll make that the user must change the browser if want something different than Google Chrome and no Internet Explorer, please. :-)

> Maybe you can also include a warning for Docker users that this will not work on Docker (Docker users will have to download the database, which is not a bid deal TBH).

How can I do this?

> BTW do we really need Selenium? Does requests.get work in this case? I guess it wouldn't work because of the CAPTCHA, is that right?

When trying to collect information from Google Cache, that's the only way that I found to extract the information and insert the captcha when needed. But when collecting directly from ConsultaSocio.com it's better with requests. I'll fix that.